essentials/src/api/dynamic.rs

```diff
@@ -113,10 +111,10 @@ fn decode_paras_entry_option(raw: &Value<u32>) -> Result<Option<ParasEntry>, Sub
 fn decode_paras_entry(raw: &Value<u32>) -> Result<ParasEntry, SubxtWrapperError> {
 	let para_id = decode_composite_u128_value(value_at("para_id", value_at("assignment", raw)?)?)? as u32;
 	let assignment = Assignment { para_id };
-	let retries = decode_u128_value(value_at("retries", raw)?)? as u32;
+	let availability_timeouts = decode_u128_value(value_at("availability_timeouts", raw)?)? as u32;
```

Comment:
I think we need to check both variants and decode what we can here.

Reply:
There hasn't been a runtime released that uses `ParasEntry` yet. I think it would be safe to assume that the `retries` field will never be seen.